get-indirection-metrics/index.js

```diff
@@ -0,0 +1,107 @@
+const { Octokit } = require("@octokit/rest");
+const fs = require("fs");
+const dig = require("node-dig-dns");
+
+const octokit = new Octokit({
+  auth: process.env.GITHUB_TOKEN,
+});
+
+const INDIRECTION_REPO = "isomerpages/isomer-indirection";
+
+async function getAllIndirectionDomains() {
+  const dnsTree = await octokit.git.getTree({
+    owner: INDIRECTION_REPO.split("/")[0],
+    repo: INDIRECTION_REPO.split("/")[1],
+    tree_sha: "main",
+    recursive: 1,
+  });
+
+  const dnsFiles = dnsTree.data.tree.filter((file) =>
+    file.path.startsWith("dns/")
+  );
+
+  const domains = dnsFiles.map((file) => file.path.split("/")[1].slice(0, -3));
```

Comment:
could we just add a quick comment on these magic numbers? i'm assuming they work but i've no idea why